pkg/http/server.go

```diff
@@ -105,19 +105,13 @@ func (s *server) useProxy(next fasthttp.RequestHandler) fasthttp.RequestHandler
 		} else {
 			proto = "http"
```

Comment:
I think we can remove useProxy entirely. we only use `http` and `Forwarded` header can be constructed by using X-forwarded-for and x-forwarded-host. so we can remove `useProxy` entirely`.

Reply:
Removed and refactored as per our discussion.